compare/compare.c

```diff
@@ -578,6 +602,16 @@ int main(int argc, char *argv[])
         scale = arg[0] - '0';
     }
 
+    int level = 2;
```

Comment:
Nit: imho writing `int level = R_BAD;` would be better

Reply:
Done.